lib/bullet.rb

```diff
@@ -147,43 +147,36 @@ def debug(title, message)
       puts "[Bullet][#{title}] #{message}" if ENV['BULLET_DEBUG'] == 'true'
     end
 
+    def thread_local_data
+      Thread.current.thread_variable_get(:bullet) || Thread.current.thread_variable_set(:bullet, {})
+    end
+
     def start_request
-      Thread.current[:bullet_start] = true
-      Thread.current[:bullet_notification_collector] = Bullet::NotificationCollector.new
+      thread_local_data[:bullet_start] = true
```

Comment:
@ildarkayumov I know it's easier to use `thread_local_data` to replace `Thread.current`, but I don't want to define the data structure `{ bullet: { bullet_start: true } }`, would you mind using the following code instead
```
Thread.current.thread_variable_set(:bullet_start, true)
```